fedimint/src/setup/configgen.rs

```diff
@@ -0,0 +1,171 @@
+use crate::config::{gen_cert_and_key, Peer as ServerPeer, ServerConfig};
```

Comment:
Add a note here that this file is pure copy-pasta that will be immediately deleted once DKG is implemented. The existing configgen script made assumptions about where configs are saved, port-numbers, etc. So wasn't usable directly. But perhaps we should update the old thing to be usable here?